apps/api-dispute/src/app/config.py

```diff
@@ -46,6 +46,9 @@ def assemble_db_connection(cls, v: Optional[str], values: dict[str, Any]) -> Any
     APP_DOMAIN: str = 'localhost'
     APP_PORT = 5000
 
+    SIGNATURE_HEADER_NAME: str = 'X-Signature'
```

Comment:
It's probably not necessary for this to be configurable, but I think it's fine for now, no need to change.